pkg/schema/objects_test.go

```diff
@@ -44,10 +44,4 @@ func TestJSONMarshal(t *testing.T) {
 	if string(b) != `"[10]int32"` {
 		t.Fail()
 	}
-
-	tc := Composite{Keys: []string{"x", "y"}, Values: []Object{Type{Name: "int32"}, Type{Name: "int32"}}}
```

Comment:
Reason for removal?

Reply:
We don't use the JSON marshaling stuff anywhere for composites (I'm actually not sure we use it for any schema object type). And it didn't work at all lol.

The JSON marshaler was just erroring out.